Module not found: Can't resolve 'label-studio' when importing label-studio frontend

I am following the [frontend reference](https://labelstud.io/guide/frontend_reference.html). I have run `npm install label-studio`. It installed successfully.

I get an error when I `import LabelStudio from "label-studio";` 

The error is `Module not found: Can't resolve 'label-studio'`. Is there another way that I should be importing label studio?

--- a/config-overrides.js
+++ b/config-overrides.js
@@ -31,5 +31,11 @@ module.exports = function override(config, env) {
     }
   }
 
+  if (process.env.BUILD_MODULE) {
+    config.output.library = "LabelStudio";
+    config.output.libraryExport = "default";
+    config.output.libraryTarget = "commonjs";
+  }
+
   return config;
 };

--- a/package.json
+++ b/package.json
@@ -72,6 +72,7 @@
     "build-notsourcemap": "GENERATE_SOURCEMAP=false npm run build",
     "build:final": "BUILD_NO_CHUNKS=true npm run build",
     "build:no-hash": "BUILD_NO_HASH=true BUILD_NO_CHUNKS=true npm run build",
+    "build:module": "BUILD_MODULE=true npm run build:no-hash",
     "build": "react-app-rewired build",
     "copy-examples": "bash scripts/copy.sh",
     "eject": "react-scripts eject",

--- a/src/index.js
+++ b/src/index.js
@@ -7,10 +7,12 @@ import App from "./components/App/App";
 import AppStore from "./stores/AppStore";
 import ProductionEnvironment from "./env/production";
 
+let LabelStudio;
+
 if (process.env.NODE_ENV === "production") {
   const environment = ProductionEnvironment;
 
-  window.LabelStudio = function(element, options) {
+  LabelStudio = function(element, options) {
     let params = options;
 
     if (params.options && params.options.secureMode) window.LS_SECURE_MODE = true;
@@ -42,7 +44,7 @@ if (process.env.NODE_ENV === "production") {
 } else {
   const environment = require("./env/development").default;
 
-  window.LabelStudio = function(element, options) {
+  LabelStudio = function(element, options) {
     let params = options;
 
     if (params.options && params.options.secureMode) window.LS_SECURE_MODE = true;
@@ -93,3 +95,7 @@ if (process.env.NODE_ENV === "production") {
     }
   };
 }
+
+window.LabelStudio = LabelStudio;
+
+export default LabelStudio;

